test/__snapshots__/TypeScriptGenerator-test.ts.snap

```diff
@@ -18,13 +18,23 @@ export type ConditionField = {
     readonly id?: string;
     readonly " $refType": "ConditionField";
 };
+export type ConditionField$data = ConditionField;
+export type ConditionField$key = {
+    readonly $data?: ConditionField$data;
+    readonly $fragmentRefs: ConditionField$ref;
```

Comment:
We don’t emit types that are named like that any more. If I understand these changes correctly, what you need here is a list of fragment references that _includes_ the fragment defined in this artefact file? If so, it should probably be `FragmentRefs<"ConditionField">`

Reply:
Also, can we add a space prefix like we did with the other keys? That makes it not show-up in auto-completion of vscode, iirc. So:

```ts
export type ConditionField$key = {
     readonly " $data"?: ConditionField$data;
     readonly " $fragmentRefs": FragmentRefs<"ConditionField">;
};
```

Reply:
Done